includes/class-accredible-learndash-event-handler.php

```diff
@@ -46,6 +46,34 @@ public static function handle_course_completed( $data ) {
 			return count( $auto_issuances );
 		}
 
+		/**
+		 * Handle `learndash_lesson_completed` Action Hooks
+		 *
+		 * @param Array $data lesson data.
+		 */
+		public static function handle_lesson_completed( $data ) {
```

Comment:
Would it be possible to extract the common logic between `handle_course_completed` and `handle_lesson_completed` to some private function for DRY?